Add initial context to README (#119)
Unleash/proxy-client-react: README.md

## Updating context

Initial context can be specified on a `FlagProvider` `config.context` property.

> `<FlagProvider config={{ ..., context: { userId: 123 }}>`

Reviewer: Don't you need to spread a named object here?

```suggestion
`<FlagProvider config={{ ...config, context: { userId: 123 }}>`
```

> Follow the steps below in order to update the unleash context dynamically:

Reviewer: Words that indicate direction (left, right, above, below, etc) don't have an inherent meaning if you can't see the text, so we should try and use words that **do** mean something regardless of direction. (Also direction depends on writing direction, so there's that too 💁🏼 ).

How about something like this? I think it's both clearer **and** it avoids indicating direction:

```suggestion
This code sample shows you how to update the unleash context dynamically:
```
